doc/rtd/explanation/index.rst

```diff
@@ -19,7 +19,7 @@ knowledge and become better at using and configuring ``cloud-init``.
    vendordata.rst
    security.rst
    analyze.rst
-   kernel-cmdline.rst
```

Comment:
Let's avoid doc file renaming throughout unless we have a really compelling reason This causes the generated docs generated to be new html pages and routes and we'd also need to setup redirect rules in RTD for each of these changes in the event of external links in blog posts etc.